definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-payment-by-account.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
I know it's ignored, but please keep it "up to date". 

Just use today's date, please.

Reply:
OK, I can see now - it was moved from other file. My bad

OK, leave it here. But for new definitions please use 'today'.

Reply:
👍